clients/admin-ui/src/features/common/Layout.tsx

```diff
@@ -21,7 +23,7 @@ const Layout = ({
     <Header />
     <NavBar />
     <main>
-      <Box px={9} py={10}>
+      <Box px={noPadding ? 0 : 9} py={noPadding ? 0 : 10}>
```

Comment:
Would it make more sense to allow the padding attribute to be configured as a prop? If added as a prop, it can default to px=8 and px=10.

Reply:
I think the padding in the Layout is otherwise pretty standard, so I thought it might make sense to instead pass a prop which tells to ignore the padding instead, but open to the suggestion or changing down the road if the layout becomes nonstandard 👀